src/posix/platform/netif.cpp

```diff
@@ -1725,6 +1725,8 @@ static void mldListenerInit(void)
     struct ipv6_mreq mreq6;
 
     sMLDMonitorFd          = SocketWithCloseExec(AF_INET6, SOCK_RAW, IPPROTO_ICMPV6, kSocketNonBlock);
+    VerifyOrDie(sMLDMonitorFd != -1, OT_EXIT_FAILURE);
```

Comment:
To fix the pretty check:
```suggestion
    sMLDMonitorFd = SocketWithCloseExec(AF_INET6, SOCK_RAW, IPPROTO_ICMPV6, kSocketNonBlock);
    VerifyOrDie(sMLDMonitorFd != -1, OT_EXIT_FAILURE);
```